youtube/generic.md

```diff
@@ -1,49 +1,57 @@
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls
 
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
+ 
+\1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+\2. Choose The Issue Type Of Invite Me To The Organization
+\3. Choose The Issue Type Of Invite Me To The Organization
+\4. Open The Issue
+\5. You get an invitation to the community by our GitHub Action 🎉
 
-## SERVICES I USE AND RECOMMEND
+---
 
-Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
-
-- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
-- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
-- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
-- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
-
-## COMMUNITY SPONSORS
+## Community Sponsors
 
-A BIG thank you to my GitHub Sponsors:
-***
 Stephen Mount http://github.com/stemount
 Nicholas Carrigan https://github.com/nhcarrigan
-***
+Santosh Yadav https://github.com/santoshyadavdev
+Andrew Cunliffe https://github.com/andrew-cunliffe
+Alexander Stoichkov https://github.com/SashoStoichkov
+Allan Regush https://github.com/AllanRegush
 
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
+---
 
-## FIND ME ON OTHER SOCIAL PLATFORMS
+## Service I Use and Recommend
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
 
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
```

Comment:
while copy pasting, all these bullets get added to the same line.
A solution might be to split them into new lines -

```suggestion
\- NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
```

This would prevent this to get weirdly formatted while copy-pasting from the markdown file:

![Screenshot 2021-06-25 at 1 26 12 PM](https://user-images.githubusercontent.com/62864373/123390917-1309af80-d5b9-11eb-979a-d413a92be8b5.png)